docs/admin/federation-apiserver.md

```diff
@@ -1,8 +1,3 @@
 This file is autogenerated, but we've stopped checking such files into the
 repository to reduce the need for rebases. Please run hack/generate-docs.sh to
 populate this file.
-
```

Comment:
This was the only line I changed manually:

https://github.com/ae6rt/kubernetes/blob/a7b6db8c55bcf0525e5cdb2c6d3ddee8879e1067/pkg/api/v1/types.go#L1311